packages/eslint-config-airbnb-base/rules/variables.js

```diff
@@ -28,7 +28,15 @@ module.exports = {
         message:
           'Use Number.isNaN instead https://github.com/airbnb/javascript#standard-library--isnan',
       },
-    ].concat(confusingBrowserGlobals),
+    ].concat(confusingBrowserGlobals.map(function (g) {
+      return {
+        name: g,
+        message:
+          "Use window." +
+          g +
+          " instead. https://github.com/facebook/create-react-app/blob/main/packages/confusing-browser-globals/README.md",
```

Comment:
this smells of prettier :-)
```suggestion
        message: "Use window." + g + " instead. https://github.com/facebook/create-react-app/blob/HEAD/packages/confusing-browser-globals/README.md",
```

Good change tho!